### PR TITLE
Recipes -- 

### PR DESCRIPTION
**Pull request by Joseph-Supples**
Recipe / View all recipes is connected to BE and displaying hardcoded data

### CODE REVIEW

File: src/components/recipes/Recipes.js [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+    let ingredientNames = [];
+    for(let j = 0; j < recipes[i].ingredients.length; j++)
+    {
+      console.log(j);
+      ingredientNames.push(recipes[i].ingredients[j].name);
+    }
+    console.log(ingredientNames.join(", "));
+
+     rows.push(createData(recipes[i].dishName, ingredientNames.join(", ")));
+  }
```
Comment (TungTNg) [inline src/components/recipes/Recipes.js:43]: Can you put this block of code either in `useEffect` or at the end of `fetch()` with `.finally()`?
I assume this block of code should only be executed after `fetch` -> `setRecipe()`?

File: src/components/recipes/Recipes.js [hunk 2]
```diff
@@ -1,7 +1,71 @@
...
+//   createData('Ice cream sandwich', ["ice cream", "bread"].join(", ")),
+//   createData('Eclair', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+//   createData('Cupcake', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+//   createData('Gingerbread', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+// ];
+
 function Recipes() {
-  return <Typography variant="h3">My Recipes page</Typography>;
+  const [recipes, setRecipes] = React.useState([]);
+  const fetchRecipes = async () => {fetch("http://localhost:3001/recipes").then((response) => response.json()).then((data) => setRecipes(data)).catch((error) => console.log(error));}
```
Comment (TungTNg) [inline src/components/recipes/Recipes.js:28]: If you're not using await & just using good old .then() chaining, no need for async here.

File: src/components/recipes/Recipes.js [hunk 3]
```diff
@@ -1,7 +1,71 @@
...
+  return { recipe, ingredients };
+}
+
+// const rows = [
+//   createData('Frozen yoghurt', ["yogurt", "milk", "sugar"].join(", ")),
+//   createData('Ice cream sandwich', ["ice cream", "bread"].join(", ")),
+//   createData('Eclair', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+//   createData('Cupcake', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+//   createData('Gingerbread', ["flour", "sugar", "butter", "eggs", "milk"].join(", ")),
+// ];
```
Comment (TungTNg) [inline src/components/recipes/Recipes.js:24]: Can you remove this stuff? The commented out block of codes

**Review by TungTNg — CHANGES_REQUESTED** [review]

File: src/components/recipes/Recipes.js (hunk 1 above)
Comment (Joseph-Supples) [reply src/components/recipes/Recipes.js]: Put in useEffect,
changed how ingredient name is handled

File: src/components/recipes/Recipes.js (hunk 2 above)
Comment (Joseph-Supples) [reply src/components/recipes/Recipes.js]: using async function and put inside of useEffect

File: src/components/recipes/Recipes.js (hunk 3 above)
Comment (Joseph-Supples) [reply src/components/recipes/Recipes.js]: Removed :)

**Review by TungTNg — APPROVED** [review]
Looks good!

**Pull request merged**